### DISCUSSION TITLE
How to send someone to Discord?

### DISCUSSION THREAD

**Discussion by cool-RR** [saulpw/visidata#1840] [Q&A] [answered]
I wanted to direct a potential contributor to our Discord, but then found that I don't know how to do that. I found no link to Discord on the site or the GitHub readme. I'm not sure that a link from my own Discord window will let him through. How do I send people to our Discord?

**Comment by saulpw** [comment, marked as answer]
https://visidata.org/chat

**Comment by anjakefala** [comment]
It's at the top of the README! https://github.com/saulpw/visidata

**Reply by cool-RR** [reply]
Ah, I see it's to the left of it. I did Ctrl-F Discord so because it's an image it wasn't found.

**Reply by saulpw** [reply]
We should also add the Discord link to our home page.  We have IRC there but not the Discord.
